benchexec/tablegenerator/react-table/src/components/Overview.js

```diff
@@ -102,16 +108,28 @@ export default class Overview extends React.Component {
 
   componentDidMount() {
     window.addEventListener("popstate", this.updateHiddenCols, false);
+    window.addEventListener("popstate", this.updateFilters, false);
   }
 
   componentWillUnmount() {
     window.removeEventListener("popstate", this.updateHiddenCols, false);
+    window.removeEventListener("popstate", this.updateFilters, false);
+  }
+
+  componentDidUpdate(_, prevState) {
+    /*     if(!deepEqual(this.state.filter, prevState.filter)){
+      this.filterPlotData(this.state.filter);
+    } */
   }
 
   updateHiddenCols = () => {
     this.setState({ hiddenCols: createHiddenColsFromURL(this.state.tools) });
   };
 
+  updateFilters = () => {
+    this.setState({ filters: getFilterParamsFromUrl() });
```

Comment:
I think there may be a small typo here. You are initializing the attribute `filter` in the constructor, but updating the attribute `filters` here in the handler - those two probably should be the same. Also this `filters` attribute here isn't used anywhere else in the code.